lib/cli/parse.js

```diff
@@ -191,6 +191,10 @@ function nodemonOption(options, arg, eatNext) {
     options.stdin = false;
   }
 
+  else if (arg === '--no-startup' || arg === '-S') {
+    options.runOnStartup = false;
```

Comment:
OK sounds good... thinking out loud here:

```
-S, --no-startup ......... disable running script on startup.

-C, --exec-on-change ......... execute script on change only, not startup
-C, --on-change-only ......... execute script on change only, not startup

-S, --no-immediate-start ......... don't execute script on startup
-S, --no-start ......... don't execute script on startup
```

I like the `--exec-on-change` or `--on-change-only` variety, partly because it makes code easier to read (i.e. the double inversion of `noStart = false` is an unnecessary brain tax).